framework/OpenMod.Core/Commands/Parameters/TimeSpanCommandParameterResolveProvider.cs

```diff
@@ -0,0 +1,25 @@
+﻿using OpenMod.API.Commands;
+using OpenMod.Core.Helpers;
+using System;
+using System.Threading.Tasks;
+
+namespace OpenMod.Core.Commands.Parameters
+{
+    public class TimeSpanCommandParameterResolveProvider : ICommandParameterResolveProvider
```

Comment:
You forgot to add `[Priority(Priority = Priority.Low)]` attribute.

Reply:
Added